src/Symfony/Bundle/FrameworkBundle/Tests/Console/Descriptor/ObjectsProvider.php

```diff
@@ -221,3 +221,24 @@ class ClassWithoutDocComment
 class ClassWithDocComment
 {
 }
+
+/**
+ * This is the first line of the description.
+ * This is the second line.
+ *
+ * This is the third and shouldn't be shown.
+ *
+ * @author Serkan Yildiz <Foo@example.com>
```

Comment:
i suggest a dummy annotation here: `@annot should not be parsed`

Reply:
Updated, thx!